src/blockdata/script.rs

```diff
@@ -315,12 +315,12 @@ impl Script {
     }
 
     /// Generates P2WPKH-type of scriptPubkey
-    pub fn new_v0_wpkh(pubkey_hash: &WPubkeyHash) -> Script {
+    pub fn new_v0_p2wpkh(pubkey_hash: &WPubkeyHash) -> Script {
         Script::new_witness_program(WitnessVersion::V0, &pubkey_hash[..])
     }
 
     /// Generates P2WSH-type of scriptPubkey with a given hash of the redeem script
-    pub fn new_v0_wsh(script_hash: &WScriptHash) -> Script {
+    pub fn new_v0_p2wsh(script_hash: &WScriptHash) -> Script {
```

Comment:
In 234d9a49aec9a272857e5a4148d4cb5a2f1a6186
nit: Deprecating change would be better because it changes the function name